container/Dockerfile.vllm

```diff
@@ -296,6 +296,7 @@ COPY --from=base --chown=$USERNAME:$USERNAME /usr/local/bin /usr/local/bin
 
 USER $USERNAME
 ENV HOME=/home/$USERNAME
+ENV PYTHONPATH=$HOME/dynamo/deploy/sdk/src:$PYTHONPATH:$HOME/dynamo/components/planner/src:$PYTHONPATH
```

Comment:
The real solution is to make planner something installed with the rest of the dynamo pip package, right? Is that work planned by anyone? CC @nv-anants @tedzhouhk 

@tanmayv25 for viz - TRTLLM container should get a similar change if we intend to test planner functionality with TRTLLM workers too.